app/src/test/java/com/realityexpander/tasky/domain/validation/IValidatePasswordTest.kt

```diff
@@ -0,0 +1,45 @@
+package com.realityexpander.tasky.domain.validation
+
+import com.realityexpander.tasky.data.validation.ValidateEmailImpl
+import com.realityexpander.tasky.data.validation.ValidatePasswordImpl
+import junit.framework.TestCase.assertFalse
+import junit.framework.TestCase.assertTrue
+import org.junit.Test
+
+
+class IValidatePasswordTest {
+
+    private val validatePassword = ValidatePasswordImpl()
```

Comment:
Why is the test class name based on the interface, but you test the implementation? Shouldn't it be `ValidatePasswordImplTest` instead`

Reply:
Good call, I wasnt sure if there was a android-specific way to do password validation, so I was leaving this open for further research.

But it looks like there isnt (im just using Regex), so I will make this a concrete implementation and remove the interface abstraction.